backend/proto/bayesian/rating.py

```diff
@@ -0,0 +1,92 @@
+import glicko2
+import math
+
+
```

Comment:
pour les classes tu devrais utiliser la synthaxe class Glicko(object) 
voir http://stackoverflow.com/questions/15374857/should-all-python-classes-extend-object